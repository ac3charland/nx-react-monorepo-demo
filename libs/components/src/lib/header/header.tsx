import { capitalizeFirstLetter } from '@utils';

export function Header() {
  return (
    <div className="flex flex-col justify-center items-center">
      <h1 className="bg-indigo-500 text-red-300 w-96 rounded p-4 mt-4 shadow-md shadow-neutral-400">
        {capitalizeFirstLetter('welcome to Nx!')}
      </h1>
    </div>
  );
}
