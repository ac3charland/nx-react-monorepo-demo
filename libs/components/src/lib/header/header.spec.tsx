import { Header } from './header';
import { render } from '@testing-library/react';

describe('Components', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<Header />);
    expect(baseElement).toBeTruthy();
  });

  it('should match snapshot', () => {
    const { baseElement } = render(<Header />);
    expect(baseElement).toMatchSnapshot();
  });
});
