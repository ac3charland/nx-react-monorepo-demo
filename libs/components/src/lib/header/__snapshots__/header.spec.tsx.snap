// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components should match snapshot 1`] = `
<body>
  <div>
    <div
      class="flex flex-col justify-center items-center"
    >
      <h1
        class="bg-indigo-500 text-red-300 w-96 rounded p-4 mt-4 shadow-md shadow-neutral-400"
      >
        Welcome to Nx!
      </h1>
    </div>
  </div>
</body>
`;
